.gitignore

```diff
@@ -11,7 +11,8 @@
 *.pydev*
 *.vcproj
 .idea/
-uid.dll
+lib/uid
+lib/uid.exe
```

Comment:
I'd say the developer should download them or a script or a binary-subrepository or something, but it's not source code...
Of course it has the drawback of not working straight away.